docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -197,136 +131,105 @@ public class CustomPropertySourceLocator implements PropertySourceLocator {
+To force a bean to re-initialize on the next method call, you must invalidate its cache entry.
+
+The `RefreshScope` is a bean in the context and has a public  `refreshAll()` method to refresh all beans in the scope by clearing the target cache.
+The `/refresh` endpoint exposes this functionality (over HTTP or JMX).
+To refresh an individual bean by name, there is also a `refresh(String)` method.
+
+NOTE: `@RefreshScope` works (technically) on an `@Configuration` class, but it might lead to surprising behavior.
+For example, it does *not* mean that all the `@Beans` defined in that class are themselves in `@RefreshScope`.
+Specifically, anything that depends on those beans cannot rely on them being updated when a refresh is initiated, unless it is itself in `@RefreshScope`.
+In that case, it is rebuilt on a refresh and its dependencies are re-injected. At that point, they are re-initialized from the refreshed `@Configuration`).
 
 === Encryption and Decryption
 
-Spring Cloud has an `Environment` pre-processor for decrypting
-property values locally. It follows the same rules as the Config
-Server, and has the same external configuration via `encrypt.\*`. Thus
-you can use encrypted values in the form `{cipher}*` and as long as
-there is a valid key then they will be decrypted before the main
-application context gets the `Environment`. To use the encryption
-features in an application you need to include Spring Security RSA in
-your classpath (Maven co-ordinates
-"org.springframework.security:spring-security-rsa") and you also need
-the full strength JCE extensions in your JVM.
+Spring Cloud has an `Environment` pre-processor for decrypting property values locally.
+It follows the same rules as the Config Server and has the same external configuration through `encrypt.\*`.
+Thus, you can use encrypted values in the form of `{cipher}*` and, as long as there is a valid key, they are decrypted before the main application context gets the `Environment` settings.
+To use the encryption features in an application, you need to include Spring Security RSA in your classpath (Maven co-ordinates: "org.springframework.security:spring-security-rsa"), and you also need the full strength JCE extensions in your JVM.
 
 include::jce.adoc[]
 
 === Endpoints
 
-For a Spring Boot Actuator application there are some additional management endpoints:
+For a Spring Boot Actuator application, some additional management endpoints are available. You can use:
 
-* POST to `/env` to update the `Environment` and rebind `@ConfigurationProperties` and log levels
-* `/refresh` for re-loading the boot strap context and refreshing the `@RefreshScope` beans
-* `/restart` for closing the `ApplicationContext` and restarting it (disabled by default)
-* `/pause` and `/resume` for calling the `Lifecycle` methods (`stop()` and `start()` on the `ApplicationContext`)
+* POST to `/env` to update the `Environment` and rebind `@ConfigurationProperties` and log levels.
```

Comment:
These are now prefixed with `/actuator`, so `/actuator/env`, same for the 4 below